packages/headless/src/api/platform-client.ts

```diff
@@ -124,76 +124,48 @@ export class PlatformClient {
   }
 }
 
-interface URLOptions<E extends PlatformEnvironment> {
-  environment?: E;
-  region?: Extract<PlatformCombination, {env: E}>['region'];
-  multiRegionSubDomain?: string;
-}
-
-function coveoCloudURL<E extends PlatformEnvironment>(
-  subdomain: string,
-  options?: URLOptions<E>
+export function getOrganizationEndpoint(
+  organizationId: string,
+  environment: PlatformEnvironment = 'prod',
+  endpointType: 'admin' | 'analytics' | 'platform' = 'platform'
```

Comment:
Curious: why not have an option for `search`?

Reply:
The way I see it, "search" is the base API URL for the search API, i.e., `<platformOrgEndpoint>/rest/search/v2`

Reply:
I might be overthinking this because not many people will use this anyway, but wouldn't it be more intuitive for users if they could access all organization endpoints, including Search, through one `getOrganizationEndpoint` function?
Asking because, as I was updating the migration docs, I was crafting an example and was surprised to see that I had to use a separate function to get the search endpoint.

Reply:
As discussed, the purpose of the `getOrganizationEndpoint` is to return the "base" endpoints (i.e., without any paths). The other exposed util functions are to get the specific API base URLs for each service (search, commerce, etc.). Mixing everything up in the same util function would be doable, but it would mean that, for instance, you'd have options that would never be used in certain contexts (e.g., there's no reason to ever get the commerce API base URL in the context of a search engine).